.github/workflows/master.yml

```diff
@@ -146,15 +146,15 @@ jobs:
           - 8
           - 11
         flink:
-          - '1.14'
           - '1.15'
+          - '1.16'
         flink-archive: [ "" ]
         comment: [ "normal" ]
         include:
           - java: 8
             flink: '1.15'
-            flink-archive: '-Dflink.archive.mirror=https://archive.apache.org/dist/flink/flink-1.14.5 -Dflink.archive.name=flink-1.14.5-bin-scala_2.12.tgz'
-            comment: 'verify-on-flink-1.14-binary'
+            flink-archive: '-Dflink.archive.mirror=https://dist.apache.org/repos/dist/dev/flink/flink-1.16.0-rc0 -Dflink.archive.name=flink-1.16.0-bin-scala_2.12.tgz'
```

Comment:
@pan3793 Are there any reasons we change to verifying forward compatibilities? I think the test is originally verifying backward compatibilities.